libdnf/transaction/private/Transaction.cpp

```diff
@@ -55,7 +58,7 @@ swdb_private::Transaction::finish(TransactionState state)
     for (auto i : getItems()) {
         if (i->getState() == TransactionItemState::UNKNOWN) {
             throw std::runtime_error(
-                tfm::format(_("TransactionItem state is not set: %s"), i->getItem()->toStr()));
+                fmt::format(_("TransactionItem state is not set: %s"), i->getItem()->toStr()));
```

Comment:
tinyformat (`tfm`) format string conforms to C `printf` format string -> `%s`
`fmt::format()` use Python-like formatting -> `{}`

Reply:
fixed